libs/auth/src/config.ts

```diff
@@ -4,6 +4,46 @@ import { isVxDev } from '@votingworks/utils';
 import { getRequiredEnvVar } from './env_vars';
 import { FileKey, TpmKey } from './keys';
 
+function shouldUseProdCerts(): boolean {
+  return process.env.NODE_ENV === 'production' && !isVxDev();
+}
+
+function getVxCertAuthorityCertPath(): string {
+  return shouldUseProdCerts()
+    ? // We can commit this prod cert to the codebase because it's 1) universal and 2) public
+      path.join(__dirname, '../certs/prod/vx-cert-authority-cert.pem')
+    : path.join(__dirname, '../certs/dev/vx-cert-authority-cert.pem');
+}
+
+function getMachineCertPathAndPrivateKey(): {
+  certPath: string;
+  privateKey: FileKey | TpmKey;
+} {
+  const machineType = getRequiredEnvVar('VX_MACHINE_TYPE');
+  const machineCertFileName =
+    machineType === 'admin'
+      ? 'vx-admin-cert-authority-cert.pem'
+      : `vx-${machineType}-cert.pem`;
```

Comment:
For reference, the non-VxAdmin machine certs are not cert authority (CA) certs (certs capable of issuing other certs), hence their different naming convention. This convention is consistent with the [create-machine-cert.sh script in vxsuite-complete-system](https://github.com/votingworks/vxsuite-complete-system/blob/a6f272bc74684807c87d80be06926550fe61ab4a/config/admin-functions/create-machine-cert.sh#L11-L15)